### PR TITLE
Create Safe spec

### PR DESCRIPTION
**Pull request by DmitryBespalov**
Only the main use case is here.

The base branch should be updated to master when the #97 is merged.

### CODE REVIEW

File: specs/create_safe/01_main.rst [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+.. step number overrides the step with the same number in the parent (inherited from) scenario.
+
+3. 1. Select "Scan" action. The camera screen opens.
+
+   2. Point the camera to the extension's QR code.
+      The message displayed that the extension connected successfully.
+
+   3. Wait until the camera closed and the app proceeds to the next step.
+
+Postconditions
```
Comment (rmeissner) [inline specs/create_safe/01_main.rst]: Shouldn't pre conditions be described first and then steps?

File: specs/create_safe/01_main.rst [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+9. Wait for the safe to be created.
+
+10. Main screen opens.
+
+.. _happy_case_postconditions:
+
+Postconditions
+++++++++++++++
+
+* New safe have been created in the blockchain.
```
Comment (rmeissner) [inline specs/create_safe/01_main.rst]: Is this really a post condition?

File: specs/create_safe/02_user_interface.rst [hunk 3]
```diff
@@ -315,15 +315,14 @@ Abort confirmation.
   <table>
     <tr>
       <td>
-        <img src="screen/android/create_safe_deposit_abort.png" width="320px"/>
+        <div width="320px>Screen not need on Android since we support multiple Safes and aborting the Safe creation is equivalent to deleting a Safe (cf. live app)</div?>
```
Comment (rmeissner) [inline specs/create_safe/02_user_interface.rst]: not needed

File: specs/create_safe/01_main.rst [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+5. Note the recovery phrase. Proceed to the next step.
+
+6. Confirm the recovery phrase. Submit your confirmation.
+   Confirmation is done by providing missing words from the phrase.
+
+7. Send required minimum transfer to the Safe address from an external account.
+   
+8. App detects new transfer on the Safe account.
+
+9. Wait for the safe to be created.
```
Comment (rmeissner) [inline specs/create_safe/01_main.rst]: capital S

File: specs/create_safe/02_user_interface.rst [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+        <img src="screens/android/create_safe_deposit.png" width="320px"/>
+      </td>
+      <td>
+        <img src="screens/ios/create_safe_deposit.png" width="320px"/>
+      </td>
+    </tr>
+  </table>
+
+* Zeplin link Android https://zpl.io/adMJrwp
+* Zeplin link iOS: https://zpl.io/bLP3GGm
```
Comment (rmeissner) [inline specs/create_safe/02_user_interface.rst]: What is the refresh button doing? It is not part of the specs

File: specs/create_safe/01_main.rst [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+
+Postconditions
+++++++++++++++
+
+- New Safe have been created on the blockchain.
+- Safe balance is what remains of the transferred funds after deducting fee for Safe creation.
+- Browser extension is connected.
+
+
+Cancellation
```
Comment (rmeissner) [inline specs/create_safe/01_main.rst]: Should we mention here too that this flow doesn't exist on Android?

Comment (tschubotz) [reply specs/create_safe/01_main.rst]: I clarified that this is only for iOS as long they don't have multiple Safes implemented. I added a scenario for Android / multiple Safes.

File: specs/create_safe/02_user_interface.rst (hunk 5 above)
Comment (tschubotz) [reply specs/create_safe/02_user_interface.rst]: Afaik the funding status of the Safe is checked. Added a note in the specs

**Pull request merged**